dislib/data/classes.py

```diff
@@ -81,7 +84,9 @@ def concatenate(self, subset, remove_duplicates=False):
         if remove_duplicates:
             self._ids, uniques = np.unique(self._ids, return_index=True)
             self.samples = self.samples[uniques]
-            self.labels = self.labels[uniques]
+
+            if self.labels is not None:
```

Comment:
isn't this the same as `if self.labels:` ?

Reply:
No, self.labels is a numpy array, so 'if self.labels' returns an array of booleans

Reply:
:+1: